plugin.json

```diff
@@ -5,7 +5,7 @@
     "homepage_url": "https://github.com/pexip/pexip-mattermost-plugin",
     "support_url": "https://github.com/pexip/pexip-mattermost-plugin/issues",
     "release_notes_url": "https://github.com/pexip/pexip-mattermost-plugin/releases/tag/v1.0.0",
-    "icon_path": "assets/starter-template-icon.svg",
+    "icon_path": "assets/icon.png",
```

Comment:
I think it would be batter to still keep this to be a vector file (eg svg) ?

Reply:
I will solve this issue and update the node version in the docs.